main.js

```diff
@@ -8,9 +8,37 @@ import { COLOR } from "./logger.js"
 import points from "./points.js";
 import { db, createTables } from "./sql.js";
 
+export const webhook = new WebhookClient({ url: config.webhook });
+
```

Comment:
make it only do that webhuk stuff when config.webhook is defined
(this also counts for the process.on listeners)

Reply:
did